### ISSUE TITLE
Issue with returning to the Extra+ menu (the one with "Go to", "Minigame", etc.)

### ISSUE THREAD

**Issue by TheShyGuy101** [zer0fixer/MAS-Extraplus#7]
Basically, when I select the "Go to" tab and then click "Nevermind", it will display an error message, forcing me to quit the game. Monika will take that as the game crashing so it isn't a big deal affection-wise, but it would still be cool if you fixed that issue
(Apparently it's only 1.0.3, 1.0.2 is unaffected)

The error message says this:

I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/Submods/ExtraPlus/Extra_Plus_Rlabels.rpy", line 20, in script
    jump return_extra
ScriptError: could not find label 'return_extra'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/Submods/ExtraPlus/Extra_Plus_Rlabels.rpy", line 20, in script
    jump return_extra
  File "J:\Doki Doki Literature Club\renpy\ast.py", line 1527, in execute
    rv = renpy.game.script.lookup(target)
  File "J:\Doki Doki Literature Club\renpy\script.py", line 858, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'return_extra'.

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Monika After Story 0.12.11

**Comment by TheShyGuy101** [comment]
update: another issue was caused by me going on a date with Monika, then leaving. After she said her leave speech, this error message occurred:

I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/Submods/ExtraPlus/Extra_Plus_Rassets.rpy", line 31, in script
    jump restore_bg
ScriptError: could not find label 'restore_bg'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/Submods/ExtraPlus/Extra_Plus_Rassets.rpy", line 31, in script
    jump restore_bg
  File "J:\Doki Doki Literature Club\renpy\ast.py", line 1527, in execute
    rv = renpy.game.script.lookup(target)
  File "J:\Doki Doki Literature Club\renpy\script.py", line 858, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'restore_bg'.

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Monika After Story 0.12.11

**Comment by zer0fixer** [comment]
> update: another issue was caused by me going on a date with Monika, then leaving. After she said her leave speech, this error message occurred:
> 
> I'm sorry, but an uncaught exception occurred.
> 
> While running game code: File "game/Submods/ExtraPlus/Extra_Plus_Rassets.rpy", line 31, in script jump restore_bg ScriptError: could not find label 'restore_bg'.
> 
> -- Full Traceback ------------------------------------------------------------
> 
> Full traceback: File "game/Submods/ExtraPlus/Extra_Plus_Rassets.rpy", line 31, in script jump restore_bg File "J:\Doki Doki Literature Club\renpy\ast.py", line 1527, in execute rv = renpy.game.script.lookup(target) File "J:\Doki Doki Literature Club\renpy\script.py", line 858, in lookup raise ScriptError("could not find label '%s'." % str(original)) ScriptError: could not find label 'restore_bg'.
> 
> Windows-8-6.2.9200 Ren'Py 6.99.12.4.2187 Monika After Story 0.12.11

Sorry about that. What you have downloaded is a contribution that I am doing with @my-otter-self that is being developed (version 1.0.4) and therefore is not finished. I will leave a link for you to download version 1.0.3.

[Google Drive] 
https://drive.google.com/drive/folders/1rDWL8a0T0lEA_VF-oYkw0oi7vwfaVG25?usp=sharing 
 
[Direct download] 
https://github.com/zer0fixer/MAS-Extraplus/archive/refs/tags/1.0.3.zip
